### PR TITLE
IBX-2640 - GraphQL filtering by attributes and content fields

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-2640 |
| Versions      | 4.1 and up |

Add filter by CFs and attributes description

### CODE REVIEW

File: docs/api/graphql_queries.md [hunk 1]
```diff
@@ -553,6 +553,79 @@ Response:
...
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+ 
+If the attribute type cannot be found in the schema, the response is:
```
Comment (konradoboza) [inline docs/api/graphql_queries.md:607]: Nitpicking: given we return two attributes, I would personally rephrase it to be more specific, something like:
```suggestion
If the attribute type cannot be found (`measure` in this case) in the schema, the response is:
```

**Review by konradoboza — APPROVED** [review]

File: docs/api/graphql_queries.md [hunk 2]
```diff
@@ -553,6 +553,79 @@ Response:
...
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+To filter products based on attributes:
+
```
Comment (juskora) [inline docs/api/graphql_queries.md:584]:
```suggestion
```

**Review by juskora — APPROVED** [review]
